index.bs

```diff
@@ -392,7 +392,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     * A [=Credential ID=].
     * A [=credential private key=].
     * The [=Relying Party Identifier=] for the [=[RP]=] that created this credential source.
-    * An optional [=user handle=] for the person who created this credential source.
+    * A [=user handle=] for the person who created this credential source.
```

Comment:
This change is already merged into master via #721, but GitHub doesn't seem to detect it because I based this PR off the `issue-720-user-handle-optional` branch whose commits were rebased onto master instead of merging.